app/routes/repositories/show/edit.js

```
@@ -40,4 +40,22 @@ export default class EditRoute extends Route {
     controller.set('clientTypeList', filteredClientTypeList);
     controller.set('clientTypes', filteredClientTypeList);
   }
+
+  setupController(controller, model) {
+    // Call _super to maintain default behavior (controller.set('model', model))
+    super.setupController(controller, model);
+
+    // Custom setup: Set additional properties
+    controller.set('isSoftwareFieldActive', false);
+  }
```

Biome (2.4.7)

[error] Duplicate class member name "setupController"

(lint/suspicious/noDuplicateClassMembers)